script.js

```diff
@@ -341,14 +409,32 @@ function skipParagraph() {
 }
 
 function render() {
-  $('html_start').style.display = display(!state.playing);
+  $('html_start').style.display = display(!state.started);
   $('language_select').style.display = display(!state.playing);
   $('html_pause').style.display = display(!state.loading && state.playing && !state.paused);
   $('html_play').style.display = display(!state.loading && state.playing && state.paused);
-  $('html_next').style.display = display(!state.loading && state.playing);
+  $('html_next').style.display = display(!state.loading && state.playing && state.started);
   $('html_skip').style.display = display(!state.loading && state.playing);
   $('html_spinner').style.display = display(state.loading);
-  $('html_title').innerHTML = state.status;
+  if (state.status) $('html_title').innerHTML = state.status;
+
+  const currectArticleSaved = state.savedArticles.find(a => a === state.currentArticle)
+  $('html_save').style.display = display(!state.loading && state.playing && !currectArticleSaved);
+  $('html_unsave').style.display = display(!state.loading && state.playing && currectArticleSaved);
```

Comment:
display/hide save/unsave button depending on the saved state of the currently playing article